inventory_management_system_api/core/config.py

```diff
@@ -17,6 +19,34 @@ class APIConfig(BaseModel):
     root_path: str = ""  # (If using a proxy) The path prefix handled by a proxy that is not seen by the app.
 
 
+class AuthenticationConfig(BaseModel):
+    """
+    Configuration model for the JWT access token authentication/authorization.
+    """
+
+    enabled: bool = False
+    public_key_path: Optional[str] = Field(default=None, validate_default=True)
+    jwt_algorithm: Optional[str] = Field(default=None, validate_default=True)
+
+    @field_validator("public_key_path", "jwt_algorithm")
+    @classmethod
+    def validate_unit(cls, field_value: str, info: ValidationInfo) -> Optional[str]:
+        """
+        Validator for the `public_key_path` and `jwt_algorithm` fields to make them mandatory if the value of the
+        `enabled` is `True`
+
+        It checks if the `enabled` field has been set to `True` and raises a `TypeError` if this is the case.
+
+        :param field_value: The value of the field.
+        :param info: Validation info from pydantic.
+        :return: The value of the field.
+        :raises ValueError: If `unit` is provided when `type` is set to `boolean`.
```

Comment:
Also this comment.

Reply:
Oops, I forgot to change this after pasting it.